ayon_api/server_api.py

```diff
@@ -148,6 +155,8 @@ def data(self):
 
     @property
     def content(self):
+        if self._response is None:
+            return None
```

Comment:
Not sure if this might be better to be an empty `bytes` or `False` instead of `None`, similar to how requests deals with it: https://github.com/psf/requests/blob/main/src/requests/models.py#L886

Reply:
```suggestion
            return b""
```